superai/apis/meta_ai/meta_ai_schema.py

```diff
@@ -0,0 +1,1140 @@
+import sgqlc.types
```

Comment:
@lpfann Can we call this file meta_ai_graphql_schema.py so that it won't be confused with the superai schema